shell/creators/pkg/init

```diff
@@ -4,11 +4,6 @@ const fs = require('fs-extra');
 const path = require('path');
 const https = require('https');
 
-const targets = {
-  dev:  './node_modules/.bin/nuxt dev',
```

Comment:
Should we be removing the `dev` script? Would it make more sense to replace it with the `vue-cli` equivalent - `"dev": "./node_modules/.bin/vue-cli-service dev",`?

Reply:
It seems like this script has never been used since we do not rely on Nuxt, or am I wrong?

Reply:
That's hard for me to say.. perhaps someone who is more familiar with the creators package can inform us. 

@jordojordo @aalves08 does it make sense to remove the `dev` script?

Reply:
(while waiting for a feedback)
If that would be the case, we would have also to restore all the

```
  "scripts": {
    "dev": "./node_modules/.bin/nuxt dev",
    "nuxt": "./node_modules/.bin/nuxt"
  },
```

As well as removing the `nuxt` dependency. But we have already stated that it's not needed (both @nwmac in chat and @codyrancher in another review)
If that would also not be the case, we have to keep `nuxt` dependency or make investigation on what means remove Nuxt globally from the extensions.

Reply:
Btw, If we cannot remove Nuxt from the extensions, then we are not able to migrate anymore in the current state and we'll have to tackle that issue. Nuxt2 is not compatible with Vue3 ecosystem.

Reply:
I don't think any of these scripts are ever used at the package level 🤔 Thoughts @jordojordo ? I don't we use any of these... It's just app-level yarn commands 🤔

Reply:
I went through the whole documentation of the extension and there's no mention on starting one, rather publishing one and that's it: https://extensions.rancher.io/extensions

Reply:
@aalves08 is correct, we don't use the package level scripts at all. Should be safe to remove those.